solidity/contracts/Chainlinked.sol

```diff
@@ -1,19 +1,27 @@
 pragma solidity ^0.4.24;
 
 import "./ChainlinkLib.sol";
+import "./ENSResolver.sol";
 import "./Oracle.sol";
+import "ens/contracts/ENS.sol";
 import "linkToken/contracts/LinkToken.sol";
 
 contract Chainlinked {
   using ChainlinkLib for ChainlinkLib.Run;
   using SafeMath for uint256;
 
-  uint256 constant clArgsVersion = 1;
+  uint256 constant internal clArgsVersion = 1;
+  uint256 constant internal linkDivisibility = 10**18;
```

Comment:
Would `linkDecimals` be a little more accurate?

Reply:
I'm open to other names, but I think "decimals" might get overloaded with the [semi-standard ERC20 decimals method of the same name](https://github.com/smartcontractkit/LinkToken/blob/master/contracts/LinkToken.sol#L12).